packages/manager/modules/core/src/secure-url/index.js

```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+import secureUrl from './secure-url.config';
+
+const moduleName = 'coreSecureUrl';
+
+angular.module(moduleName, []).run(secureUrl);
```

Comment:
**question:** As `$transitions` provider is used, shouldn't `@uirouter/angularjs` module be injected as a module dependency?